### PR TITLE
deprecated get stacktrace

### PR DESCRIPTION
**Pull request by getong**

### CODE REVIEW

File: include/exometer.hrl [hunk 1]
```diff
@@ -36,3 +36,11 @@
 -record(exometer_shared, {
           name
          }).
+
+-ifdef(OTP_RELEASE). %% this implies 21 or higher
```
Comment (michalwski) [inline include/exometer.hrl:40]: Where is this `OTP_RELEASE` set? I would expect it to be set somewhere in rebar.config when we detect we run on Erlang/OTP 21 or higher.

**Comment by getong** [comment]
http://www.erlang.org/news/123
```
A new predifined macro `OTP_RELEASE`
```
@michalwski

**Comment by michalwski** [comment]
Thanks for explanation. Could you also update the PR description so that you summarise a bit what are the changes introduced in the PR?

**Comment by getong** [comment]
@michalwski New msg updated.

**Comment by getong** [comment]
The upstream is changed, I think you should keep it updated with the upstream.

**Pull request closed without merging**